test/uk/gov/hmrc/plasticpackagingtax/registration/views/ConfirmationViewSpec.scala

```diff
@@ -70,20 +72,20 @@ class ConfirmationViewSpec extends UnitViewSpec with Matchers {
     "display panel" when {
 
       "no 'referenceId' has been provided" in {
-        view.getElementsByClass("govuk-panel__title").get(0) must containMessage(
+        view.getElementsByClass(Styles.gdsPanelTitle).get(0) must containMessage(
```

Comment:
Can drop the `Styles.` with the implicit import

Reply:
Yeah, I know. I prefer the context that it gives but am happy to be consistent.